types/block.go

```diff
@@ -14,15 +14,15 @@ import (
 	"github.com/tendermint/tmlibs/merkle"
 )
 
-// Block defines the atomic unit of a Tendermint blockchain
+// Block defines the atomic unit of a Tendermint blockchain.
 type Block struct {
 	*Header    `json:"header"`
 	*Data      `json:"data"`
 	LastCommit *Commit `json:"last_commit"`
 }
 
-// MakeBlock returns a new block and corresponding part set from the given information
-// TODO: version
+// MakeBlock returns a new block and corresponding part set from the given information.
+// TODO: Add version information to the Block struct.
```

Comment:
I think @zramsay got us covered with this in https://github.com/tendermint/tendermint/pull/736 and in particular https://github.com/tendermint/tendermint/pull/736/files#diff-fd289e47592d409909487becb9d38925R38.

Reply:
Yes, but depending on which one gets merged first let's keep it here. If we remove it here and merge it and then the second one doesn't get merged we would lose the todo.